app/templates/view_supplier_users.html

```diff
@@ -156,24 +151,17 @@
           <div class="grid-row">
               <div class="column-two-thirds">
                   <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
-                  {% if invite_form.email_address.errors %}
-                  <div class="validation-wrapper">
-                      {% endif %}
-                      <div class="question">
-                          {{ invite_form.email_address.label(class="question-heading") }}
-                          <p class="hint">
-                              Enter the email address of the person you wish to invite
-                          </p>
-                          {% if invite_form.email_address.errors %}
-                          <p class="validation-message" id="error-email-address-textbox">
-                              {% for error in invite_form.email_address.errors %}{{ error }}{% endfor %}
-                          </p>
-                          {% endif %}
-                          {{ invite_form.email_address(class="text-box", autocomplete="off") }}
-                      </div>
-                      {% if invite_form.email_address.errors %}
-                  </div>
-                  {% endif %}
+                  {%
+                    with
+                    question = invite_form.email_address.label,
+                    hint = "Enter the email address of the person you wish to invite",
+                    name = invite_form.email_address.name,
+                    value = invite_form.email_address.data,
+                    error = invite_form.email_address.errors[0]
```

Comment:
This might be a daft question but will `.errors[0]` break if there are no errors here? Or does the toolkit template deal with it for us?

Reply:
It's definitely fine - tried and tested - Jinja just carries on if it gets a None for things like this.

Reply:
Phew - I do get overly paranoid about `[0]` sometimes...

Reply:
Me too - hence I know from recent experience it's fine.
I was looking for a neater way to do it but couldn't come up with anything 😞  - open to suggestions if anything springs to mind, but I think this is fine if not.